tests/functional/serving/test_experiments_endpoint.py

```diff
@@ -75,13 +76,14 @@ def test_send_name_and_versions(self, client):
     def test_latest_versions(self, client):
         """Tests that the API return the latest versions of each experiment"""
         expected = [
-            {'name': 'a', 'version': 2},
+            {'name': 'a', 'version': 3},
             {'name': 'b', 'version': 1}
         ]
 
         _add_experiment(name='a', version=1, _id=1)
+        _add_experiment(name='a', version=3, _id=3)
```

Comment:
I would swap the ids to avoid having the DB reshuffle the data according to ID. So (version=3, _id=2), (version=2, _id=3).